R/system.R

```diff
@@ -768,21 +767,15 @@ getTwitter <- function(n=200, lang=NULL,  lastNDays=30, searchString, tokenFileI
 
   # convert search string to UTF-8 before sending it on the wire on windows.
   searchString <- convertUserInputToUtf8(searchString)
-  tweetList <- twitteR::searchTwitter(searchString, n, lang, since, until, locale, geocode, sinceID, maxID, resultType, retryOnRateLimit)
-  # conver list to data frame
-  if(length(tweetList)>0){
-    ret <- twitteR::twListToDF(tweetList)
-    if(withSentiment){
-      # calculate sentiment
-      ret %>% dplyr::mutate(sentiment = get_sentiment(text))
-    } else {
-      ret
-    }
+  tweetList <- rtweet::search_tweets(q = searchString, token = twitter_token, n = n, lang = lang, verbose = TRUE, since = since, unitl = until, locale = locale, geocode = geocode, type = resultType,  retryonratelimit=TRUE)
+  if(withSentiment){
```

Comment:
Looks the handling of "no tweets found" case is removed, but is it ok? if so why?